src/styles/main.scss

```diff
@@ -1 +1,62 @@
-// put code here
+@import "./utils/general.scss";
+@import "./utils/varibales.scss";
+
+.watch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 10px dashed #000;
+  border-radius: $watch-radius;
+  margin: 0 auto;
```

Comment:
- create a container for your watch, and use BEM element to set position-relative for your watch. It's not the best option position BEM block
![Annotation 2023-09-06 230002](https://github.com/mate-academy/layout_stop-watch/assets/109142598/01b1a414-142d-49bc-83fb-906679fd01a6)

- remove margin,  you can center the watch with flex